### ISSUE TITLE
createUUID causing slowness under times of extreme I/O

### ISSUE THREAD

**Issue by bockensm** [jcberquist/function-linenums#2]
We have a restart cycle for our servers which host dozens of even hundreds of ColdFusion applications. During that time we get errors that take quite a while to report to Sentry. I used FusionReactor's Profiler to try to diagnose what's going on during that time and found that the `createUUID()` function used in this library is waiting on a lock that takes a while to free up. I was looking through the code to see what `createUUID()` was doing and if we could replace it with something else.

Is there any harm in replacing that with something non-blocking or does it have to specifically be a UUID? I'm also curious if using a `java.util.UUID` would yield the same blocking behavior.

**Comment by bockensm** [comment]
Here's what the profiler shows:

![fr-profiler](https://user-images.githubusercontent.com/2660751/123997101-a0704980-d995-11eb-991f-dad51198e09a.png)

**Comment by jcberquist** [comment]
Hi @bockensm, I am sorry but I totally missed this issue back when you raised it, it seems I had managed to turn notifications off for this repo. Is this still an issue for you?

**Comment by bockensm** [comment]
I haven't noticed this so much lately, but I would still like to have the change considered. That ID doesn't seem to contribute anything meaningful if I'm reading the code correctly.

**Comment by jcberquist** [comment]
Thanks for the update. I would like to keep that id in there for now, I use it here [functionLineNums.cfc#L268](https://github.com/jcberquist/function-linenums/blob/master/functionLineNums.cfc#L268) to check object (struct) identity. When I tried to use `===` instead, I had stack overflow issues. I think those issues have since been fixed, but I would still like to stay away from that for now.

Unfortunately I am not sure that going to `java.util.UUID` will fix it as it looks to me like the lucee method just defers to it: [CreateUUID.java](https://github.com/lucee/Lucee/blob/541388c2d8b392b5d77fe6fbb766fa2fa104950c/core/src/main/java/lucee/runtime/functions/other/CreateUUID.java). But I will poke around more and see if there are other ways to use it that avoid the locking.
